web-ng/src/app/shared/models/project.model.ts

```diff
@@ -27,7 +27,7 @@ export class Project {
     readonly layers: Map<string, Layer>
   ) {}
 
-  getLayerForm(layerId: string, formId: string): Form {
+  getForm(layerId: string, formId: string): Form {
```

Comment:
Also, not sure we want to throw an error if not found directly in the model. In general the model accessors return `null` for unknown members, so perhaps should do the same here for consistency?